domain.yml

```diff
@@ -25,43 +24,71 @@ slots:
       - type: from_llm
   validity_extension_offer:
     type: text
+    mappings:
+      - type: custom
+        action: action_fetch_tender_details
   project_name:
     type: text
+    mappings:
+      - type: custom
+        action: action_fetch_tender_details
   project_id:
     type: text
     mappings:
       - type: custom
         action: action_fetch_tender_details
   tender_type:
     type: text
+    mappings:
+      - type: custom
+        action: action_fetch_tender_details
   selected_hotels:
     type: list
+    mappings:
+      - type: from_llm
   tender_confirmation:
     type: bool
-    influence_conversation: true
-    initial_value: false
+    mappings:
+      - type: from_llm
   final_confirmation:
     type: bool
     influence_conversation: true
-    initial_value: false
+    initial_value: null
   selected_hotels_contacts:
     type: text
+    mappings:
+     - type: from_llm
+  ask_new_date:
+    type: text
+    mappings:
+     - type: from_llm
+  date_confirmation:
+    type: text
+    mappings:
+     - type: from_llm
+  user_hotel_selection:
+    type: string
```

Comment:
@rams0413 i get an error: "Failed to find slot type, 'string' is neither a known type nor user-defined."